designs/local-pv/lvm/storageclass-parameters/fs_type.md

```diff
@@ -78,7 +78,7 @@ provisioner: local.csi.openebs.io
 parameters:
   storage: "lvm"
   vgpattern: "lvmvg.*"
-  fsType: xfs 
+  fsType: xfs
```

Comment:
should it have quotes like the ones above?

Reply:
It's not mandatory. But will add it to keep it similar.